File: Pokua/Exercise27.java
import java.util.HashMap;

public class Exercise27 {
    public static void  main(String[] args){
        HashMap<String, Integer> hashMap = new HashMap<String, Integer>();

        hashMap.put("Harry",81);
        hashMap.put("Ron",78);
        hashMap.put("Hermione",99);
        hashMap.put("Draco", 74);
        hashMap.put("Neville", 62);

        HashMap<String, Integer> ratings = new HashMap<String, Integer>();

        for(int i: hashMap.values()){
            if (i>=91 && i<=100){
                ratings.put("Outstanding",i);
            }else if (i>=81 && i<=90){
                ratings.put("Exceeds Expectation",i);
            }else if(i>=71 && i<=80){
                ratings.put("Acceptable",i);
            }else {
                ratings.put("Fail",i);
            }
        }

    }
}

File: Stephannie/Exercise-27.py
#city visits
scores_data = {
    "Harry": 81,
    "Ron": 78,
    "Hermione": 99,
    "Draco": 74,
    "Neville": 62,
}

def ratings(score):
    if score >= 91:
        return "Outstanding"
    elif score >= 81:
        return "Exceeds expectation"
    elif score >= 71:
        return "Acceptable"
    else:
        return "Fail"


for key in scores_data:
    scores_data[key] = ratings(scores_data[key])

      








